environments/demo/docker-compose-traefik-latest.yml

```diff
@@ -1,75 +0,0 @@
-#-------------------------------------------------------------------------------
```

Comment:
docker-compose.yml already get the latest images. If you need both I can add it.